packages/@okta/vuepress-site/docs/concepts/ie-intro/index.md

```diff
@@ -68,10 +64,10 @@ Identity engine introduces new concepts and terminology.
 
   App sign-on policies enforce authentication when an end user attempts to access an application. Each application is assigned one app sign-on policy (with multiple rules that you can order). See [App sign-on policies](https://help.okta.com/en/oie/Content/Topics/identity-engine/policies/about-app-sign-on-policies.htm).
 
-## Enable Identity engine for your organization
+## Enable Identity Engine for your organization
 
-To upgrade to Identity engine, reach out to your account manager. If you don't have an account manager, reach out to oie@okta.com for more information.
+To upgrade to Identity Engine, reach out to your account manager. If you don't have an account manager, reach out to oie@okta.com for more information.
```

Comment:
Not a fan of "reaching out" here. How 'bout:

`To upgrade to Identity Engine, contact your account manager. If you don't have an account manager, email oie@okta.com for more information.`